src/index.html

```diff
@@ -1,10 +1,11 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8" />
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
But we are adding it via resolvers, aren't we?

Reply:
Nope, we removed it for viewport. There is a glitch if we are adding it via js. It's working with SSR, but we don't have it for FA.